coordinape/coordinape
# User Logout hasura

### Test Plan

1. Find out the `id` of the profile you are using to authenticate with hasura
2. Go to `personal_access_tokens` and search the `id` against the `tokenable_id` field
3. This mutation will purge your access token for hasura so you might want to copy down the data of the row first to insert back later
4. Execute the the mutation 
```gql
mutation MyMutation {
  logout_user {
    profile {
      address
      id
    }
  }
}
```
5.  Verify you cannot query hasura with your existing token anymore, you can access your DB to check the row is indeed deleted.
6. Add back your token and make sure you can access hasura again.

Closes #563
